client/src/components/EventPage/EventPage.js

```diff
@@ -135,39 +172,161 @@ class EventPage extends Component {
+                    id: 12,
+                    name: 'Davion Dennis',
+                    image: 'http://i.pravatar.cc/36',
+                    joined: '2 days ago'
+                },
+                {
+                    id: 13,
+                    name: 'Remington Dalton',
+                    image: 'http://i.pravatar.cc/36',
+                    joined: '1 day ago'
+                },
+                {
+                    id: 14,
+                    name: 'Direct Elton',
+                    image: 'http://i.pravatar.cc/36',
+                    joined: '12 days ago'
+                }
+            ],
+            gallery: [
+                {
+                  src: "http://en.es-static.us/upl/2018/12/meteor-geminids-venus-12-15-2018-Kota-Belud-Sabah-Emma-Zulaiha-Zulkifli-e1545048674227.jpg",
+                  thumbnail: "http://en.es-static.us/upl/2018/12/meteor-geminids-venus-12-15-2018-Kota-Belud-Sabah-Emma-Zulaiha-Zulkifli-e1545048674227.jpg",
+                  thumbnailWidth: 520,
+                  thumbnailHeight: 274
+                },
+                {
+                  src: "https://bloximages.chicago2.vip.townnews.com/postregister.com/content/tncms/assets/v3/editorial/4/82/482ee1c8-a447-5c16-a0e4-b350d77faea2/5b6dcccf942ce.image.jpg?resize=400%2C600",
+                  thumbnail: "https://bloximages.chicago2.vip.townnews.com/postregister.com/content/tncms/assets/v3/editorial/4/82/482ee1c8-a447-5c16-a0e4-b350d77faea2/5b6dcccf942ce.image.jpg?resize=400%2C600",
+                  thumbnailWidth: 320,
+                  thumbnailHeight: 512,
+                  tags: [{value: "Rivne", title: "Rivne"}],
+                },
+                {
+                  src: "https://texashillcountry.com/wp-content/uploads/Meteor-Shower.jpg",
+                  thumbnail: "https://texashillcountry.com/wp-content/uploads/Meteor-Shower.jpg",
+                  thumbnailWidth: 450,
+                  thumbnailHeight: 260
+                },
+                {
+                  src: "https://r.hswstatic.com/w_907/gif/leonidmeteors-1.jpg",
+                  thumbnail: "https://r.hswstatic.com/w_907/gif/leonidmeteors-1.jpg",
+                  thumbnailWidth: 320,
+                  thumbnailHeight: 430,
+                  tags: [{value: "Rivne", title: "Rivne"}]
+                },
+                {
+                  src: "https://cdn-az.allevents.in/banners/85c41d80-b181-11e8-81c9-1b431fd718bc-rimg-w400-h400-dc374665-gmir.jpg",
+                  thumbnail: "https://cdn-az.allevents.in/banners/85c41d80-b181-11e8-81c9-1b431fd718bc-rimg-w400-h400-dc374665-gmir.jpg",
+                  thumbnailWidth: 620,
+                  thumbnailHeight: 374
                 }
             ]
         }
 
+        const _this = this;
+        
         // @temp, immitate delay;
         window.setTimeout(() => {
-            this.setState(() => data); 
-        }, 500);
+            this.setState({...data});
+        }, 500);   
     }
```

Comment:
This will work like your code, but (I think) more easier for reading. And You can move your mock ( const data=... ) over your class.
componentDidMount = () => {
         // @temp, immitate delay;
 setTimeout(() => this.setState({data}), 500);
}

Reply:
Yes, agree. Issue is I will remove that part, hopefully today, if health ok. That is why I left it that way.